src/plugins/intel_npu/tools/single-image-test/main.cpp

```diff
@@ -95,6 +95,7 @@ DEFINE_string(
 DEFINE_string(data_shape, "",
     "Required for models with dynamic shapes. Set shape for input blobs. Only one shape can be set."
     "In case of one input size: \"[1,3,224,224]\"");
+DEFINE_string(skip_output_layers, "" , "Skip output layers from the network");
```

Comment:
probably we'd better clarify that is it for tests like RRMSE & NRMSE  and a separation for layers is ';'

Reply:
Resolved. Thanks!